src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html

```diff
@@ -38,17 +38,17 @@
     <h2 class="text-muted">
       <i class="fas fa-trash-alt"></i> Feedback Sessions Recycle Bin
     </h2>
-    <div class="card">
-      <div class="card-header recycle-bin-header" (click)="isRecycleBinExpanded = !isRecycleBinExpanded">
+    <div class="card bg-light top-padded">
+      <div class="card-header bg-secondary recycle-bin-header" (click)="isRecycleBinExpanded = !isRecycleBinExpanded">
         <div class="row align-items-center">
           <div class="col-6 text-white">
             <b>Recycle Bin</b>
           </div>
-          <div class="col-6 text-right">
+          <div class="col-6 text-right actions-cell">
```

Comment:
Perhaps a better name would be `bin-header-button`? `actions-cell` refers to this:
![image](https://user-images.githubusercontent.com/35655790/55666760-da6d4680-5885-11e9-80c5-fc97790f9024.png)